_sass/_libs/wds/stylesheets/components/_accordions.scss

```diff
@@ -4,70 +4,99 @@ $accordion-border: 3px solid $color-gray-lightest;
 
 // Accordion Styles
 
+@mixin accordion-list-styles {
```

Comment:
Follow up question, @elainekamlley. Are all the changes to the wds stylesheets simply updates to wds 0.11.0? 

Cause then I take back my comments and we should merge this.